BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -50,8 +54,8 @@ extension ElementContent {
 }
 
 extension ElementContent {
-    public init(build builder: @escaping (Environment) -> Element) {
-        self.storage = LazyStorage(builder: builder)
+    public static func withEnvironment(_ builder: @escaping (Environment) -> Element) -> ElementContent {
```

Comment:
Swapped these out b/c the two init methods that take closures cause conflicts which confuse the compiler. Pushing them behind a static func makes the intention clearer and prevents future issues.